src/frontend/components/LiveInfoBar/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { useParticipantsStore } from 'data/stores/useParticipantsStore';
+import { Box, Heading, Paragraph } from 'grommet';
+import React, { Fragment } from 'react';
+import { DateTime } from 'luxon';
+import { Nullable } from 'utils/types';
+import { useIntl } from 'react-intl';
+
+interface LiveInfoBarProps {
+  title: string;
+  startDate: Nullable<string>;
+}
+
+export const LiveInfoBar = ({ title, startDate }: LiveInfoBarProps) => {
+  const participants = useParticipantsStore((state) => state.participants);
+  const intl = useIntl();
+
+  if (startDate) {
+    const dt = DateTime.fromISO(startDate);
+    startDate = dt.isValid
```

Comment:
you should not override `startDate` but create a new variable. This is error prone.

Reply:
And I don't know if it should not be managed with a useMemo. There is no need to compute this at every render.

Reply:
Usually I use useMemo when the computation seems a bit heavy, but yes we can do it.

Reply:
If it's not then just change the variable name.